using_the_compiler/README.md

```diff
@@ -44,6 +44,12 @@ Make sure to always use `--release` for production-ready executables and when pe
 
 The reason for this is that performance without full optimizations is still pretty good and provides fast compile times, so you can use the `crystal` command almost as if it were an interpreter.
 
+## Reducing the executable's binary size
+
+This will remove debug symbols. Use this options only if the size is an issue and debugging isn't planned.
```

Comment:
Can you send this paragraph below the one that starts with "To reduce...", it's often best to start with the core of the section ("to reduce binary size you can you the `--no-debug` flag") and then explain further ("you get a smaller binary file because you're effectively removing debug symbols").

Reply:
"this options" => "these options"